CMakeLists.txt

```diff
@@ -36,9 +35,10 @@ endif()
 # for Linux, BSD, Solaris, Minix
 if(UNIX AND NOT APPLE)
     SET(Qt5Widgets_DIR "/usr/include/x86_64-linux-gnu/qt5/QtWidgets")
+    SET(Qt5Charts_DIR "/usr/include/x86_64-linux-gnu/qt5/QtCharts")
 endif()
 
-
+find_package(Qt5Charts REQUIRED)
```

Comment:
Does this require a new dependency? Do we need to update the installation guide?

Reply:
No this dependency is already in development. As you can probably see, this PR still has conflicts with the base branch. (it needs another PR to be merged first, and i suspect most conflicts will be gone by then)